lib/kafka/protocol/fetch_response.rb

```diff
@@ -38,11 +38,14 @@ def initialize(name:, partitions:)
 
       attr_reader :topics
 
-      def initialize(topics: [])
+      def initialize(topics: [], throttle_time_ms: 0)
```

Comment:
is the ```throttle_time_ms``` going to be configurable?

Reply:
It's not set by the client, but rather by the broker. From [the docs](https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-FetchResponse):

> Duration in milliseconds for which the request was throttled due to quota violation. (Zero if the request did not violate any quota.)

Reply:
I don't have any plans to add support for quotas, but I'm totally open to PRs :)